src/utils/use-transportation-color.ts

```diff
@@ -46,3 +47,24 @@ export const useThemeColorForTransportMode = (
       return 'transport_other';
   }
 };
+
+export function useTransportationLineColors(fareProductType?: string) {
```

Comment:
Now when the line color is fixed I think it doesn't need to be returned in this hook, it can just be in the styles directly. And then maybe the hook is unnecessary, and `useTransportationColor` can be used instead? If the code is clearer with a hook I think I would prefer a name like `useTransportationColorForFareProductType`.

Reply:
I let the name names because it refers to that animated lines:

![image](https://github.com/AtB-AS/mittatb-app/assets/43450882/1548b5e2-3dd5-42d5-9fe4-bdfaf6a2b623)

Reply:
Yeah, I kinda figured, but think it creates a to strong coupling between the specific `ValidityLine` component and a shared hook under `src/utils`. If the hook is specific for `ValidityLine`, and named accordingly, it should instead be under `src/fare-contracts`.

Reply:
I made a refactor based on your comments ;)